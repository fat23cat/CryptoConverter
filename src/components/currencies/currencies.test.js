import React from 'react';
import ReactDOM from 'react-dom';
import Currencies from './currencies.js';

describe('components', () => {
  it('should render a Currencies', () => {
   const wrapper = (<Currencies />);
   expect(wrapper).toMatchSnapshot();
  })
})
