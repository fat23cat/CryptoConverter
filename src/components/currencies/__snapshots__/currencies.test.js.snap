// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components should render a Currencies 1`] = `<Connect(Currencies) />`;
